### PR TITLE
Fix unhandled errors and changed to camel case where applicable

### PR DESCRIPTION
**Pull request by leo8a**
Basically, added code for some unhandled errors as well as changed to camel case instead of snake case in variables across the code.

Signed-off-by: Leo Ochoa [lochoa@redhat.com](mailto:lochoa@redhat.com)

### CODE REVIEW

**Comment by alknopfler** [comment]
@leo8a re-check it out again because with the last refactor has been changing the error handling again

**Comment by alknopfler** [comment]
@leo8a rebase please ;)

**Comment by leo8a** [comment]
refactor looks great, thanks @alknopfler and @karmab!

File: pkg/operator/info.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 func (o *Operator) GetInfo(args []string) error {
 	if len(args) != 1 {
 		color.Set(color.FgRed)
-		return errors.New("Invalid number of arguments. Usage: tasty info OPERATOR_NAME")
```
Comment (karmab) [inline pkg/operator/info.go:18]: Why not start a sentence with a capital letter?

Comment (leo8a) [reply pkg/operator/info.go:18]: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

**Pull request merged**